tests/kernel/update_kernel.pm

```diff
@@ -30,7 +31,8 @@ sub check_kernel_package {
     # Only check versioned kernels in livepatch tests. Some old kernel
     # packages install /boot/vmlinux symlink but don't set package ownership.
     my $glob = get_var('KGRAFT', 0) ? '-*' : '*';
-    my $cmd = 'rpm -qf --qf "%{NAME}\n" /boot/vmlinu[xz]' . $glob;
+    my $run_cmd = is_transactional ? 'transactional-update -c -d --quiet run ' : '';
+    my $cmd = $run_cmd . 'rpm -qf --qf "%{NAME}\n" /boot/vmlinu[xz]' . $glob;
```

Comment:
Not optimal solution, already done in `install _ltp`. Needs separate function.

Reply:
This modifications are not needed if is system reboot just after the patch. I decided to go with addition reboot to not increase complexity of test code by wrappers.